### DISCUSSION TITLE
best VPN protocol

### DISCUSSION THREAD

**Discussion by mahansalimi** [HirbodBehnam/MTProtoProxyCentOSInstall#59] [Q&A]
Hello 
already i was using wireguard Protocol ,but seems goverment baned all protocol
what can we do?is there any other protocol that you can suggest?
* MTproto is working well yet.
also outline and openvpn and openconnect  dont work

**Comment by HirbodBehnam** [comment]
Hello
I personally use Trojan + TLS + Http2. It works pretty well IMO. I use xray as server and client.

**Comment by mahansalimi** [comment]
hello Mr Hirbood
could you please help me to enable http2? i was using Trojan+TLS until now.recently has problem in Irancell.
with http2 is working well yet?
or if has proble, which protocol you suggest?
